kuma/urls.py

```diff
@@ -1,6 +1,7 @@
 from decorator_include import decorator_include
 from django.conf import settings
 from django.conf.urls import include, url
+# from django.conf.urls.i18n import i18n_patterns
```

Comment:
Can this be removed, or is it a reminder?

Reply:
Nope, it can be removed. I tried many, many things, and didn't clean up completely 😬